.golangci.yml

```diff
@@ -7,7 +7,7 @@ run:
 # output configuration options
 output:
   # colored-line-number|line-number|json|tab|checkstyle|code-climate, default is "colored-line-number"
-  format: colored-line-number
+  formats: colored-line-number
```

Comment:
typo? `s` seems wrong here
look at https://golangci-lint.run/usage/configuration/

Reply:
![image](https://github.com/opiproject/sztp/assets/82646284/00c68782-3d81-435f-961f-2157037843f0)

output.format was used in older version golangci-lint
it was changed to output.formats in the later versions

Reply:
format was used in older versions
it is still supported but yea gives a warning

i actually made a mistake here it shoud have been : 
 ```
 formats:
       - format: colored-line-number
         path: stdout
         
```

Reply:
i'll change it and push

Reply:
thanks